<?xml version="1.0"?>
<launch>
    <arg name="publish_wave" default="false" />

    <node pkg="snowboy_ros" type="hotword_detector_node" name="snowboy" required="true" output="screen">
        <param name="resource_filename" value="$(find snowboy_ros)/resources/common.res"/>
        <param name="model_filename" value="$(find snowboy_ros)/resources/HiAqua.pmdl"/>
        <param name="sensitivity_str" value="0.5" type="str" />
        <param name="audio_gain" value="1.0"/>
        <param name="publish_wave" value="$(arg publish_wave)" />
    </node>
</launch>
